packages/lesswrong/components/tagging/TagPage.tsx

```diff
@@ -321,6 +321,7 @@ const TagPage = ({classes}: {
           </div>
           <TagPageButtonRow tag={tag} editing={editing} setEditing={setEditing} className={classNames(classes.editMenu, classes.nonMobileButtonRow)} />
         </div>}
+        welcomeBox={null}
```

Comment:
move this above the tableOfContents so it's easier to parse.

Reply:
Extracted all the other ToCColumn props above the return to make them easier to read (and left them "in order")